### PR TITLE
added docs for envis_aggregator.py

### PR DESCRIPTION
**Pull request by bashirk**
## Description
This PR adds inline documentation to the envis_aggregator.py file.

## Relevant Issue
Issue #269 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)

### CODE REVIEW

File: fedrec/user_modules/envis_aggregator.py [hunk 1]
```diff
@@ -3,7 +3,18 @@
 
 class EnvisAggregator(EnvisBase):
     """
-    Class for aggregating Envis.
+    This class is used to aggregate the results of multiple EnvisBase modules.
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_aggregator.py:6]:
```suggestion
    EnvisAggregator is used to aggregate the results of multiple EnvisBase modules.
```

File: fedrec/user_modules/envis_aggregator.py [hunk 2]
```diff
@@ -3,7 +3,18 @@
...
+    By aggregating, we mean that we can combine the results of multiple modules
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_aggregator.py:7]: You can rewrite it the `we mean that we can combine` as we combine

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/user_modules/envis_aggregator.py [hunk 3]
```diff
@@ -3,7 +3,19 @@
...
+    EnvisBase modules. By aggregating, we combine the results of
+    multiple modules into a single result. For example, we can
+    aggregate the results of multiple modules that are trained on
+    the same data set. We can also aggregate the results of
+    multiple modules that are trained on different data sets.
+
+    Methods
+    -------
+    __call__():
+        Aggregate the results of the modules.
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_aggregator.py:16]: You can remove it as in it's not implemented.

File: fedrec/user_modules/envis_aggregator.py [hunk 4]
```diff
@@ -3,7 +3,19 @@
 
 class EnvisAggregator(EnvisBase):
     """
-    Class for aggregating Envis.
+    EnvisAggregator is used to aggregate the results of multiple
+    EnvisBase modules. By aggregating, we combine the results of
+    multiple modules into a single result. For example, we can
+    aggregate the results of multiple modules that are trained on
+    the same data set. We can also aggregate the results of
+    multiple modules that are trained on different data sets.
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_aggregator.py:11]: It's not making any sense. You're just repeating the same thing again and again. Please be precise.
Thanks

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/user_modules/envis_aggregator.py [hunk 5]
```diff
@@ -3,7 +3,18 @@
 
 class EnvisAggregator(EnvisBase):
     """
-    Class for aggregating Envis.
+    The EnvisAggregator class is in charge of aggregating the data from
+    the workers. It is used to aggregate the results of multiple EnvisBase
+    modules into a single result.
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_aggregator.py:8]: `It is used to aggregate the results of multiple EnvisBase modules into a single result.` rephrase this part

File: fedrec/user_modules/envis_aggregator.py [hunk 6]
```diff
@@ -3,7 +3,18 @@
...
+    
+    For example, we can aggregate the results of multiple modules that are
+    trained on the same data set, and also the results of those that are
+    trained on different data sets.
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_aggregator.py:12]: If you want to add an example, add code snippets to perform this.

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
LGTM!! @ramesht007

**Review by ramesht007 — APPROVED** [review]
LGTM!!

**Pull request merged**